### PR TITLE
Add favicon

### PR DESCRIPTION
**Pull request by jameseaster**
#### _Resolves - add favicon_

---

### What kind of change does this PR introduce?

- Style update.

<br/>

### What is the current behavior?

- No favicon.

<br/>

### What is the new behavior?

- Favicon with the initials VC.

<br/>

### Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

<br/>

### How has this been tested?

- Locally rendered.

<br/>

### CODE REVIEW

**Review by BigBlueHat — CHANGES_REQUESTED** [review]
We may as well name the site in the manifest, so if someone "installs" it, it gets a name.

File: images/favicon/site.webmanifest
Comment (davidlehn) [inline images/favicon/site.webmanifest:1]: - This is a strange source location for this file.
- Would be easier to read if indented.

File: .eleventy.js [hunk 1]
```diff
@@ -71,10 +71,12 @@ module.exports = function (eleventyConfig) {
     return str.slice(0, endIdx);
   });
 
+  // Copy favicon files to root
+  eleventyConfig.addPassthroughCopy({ "images/favicon" : "/" });
   eleventyConfig.addPassthroughCopy("assets/");
   eleventyConfig.addPassthroughCopy("**/*.jpg");
   eleventyConfig.addPassthroughCopy("**/*.png");
-
+
```
Comment (davidlehn) [inline .eleventy.js:79]:
```suggestion

```

**Review by davidlehn — COMMENTED** [review]
- The white on light blue contrast makes it more difficult to read at small sizes, especially with the light upper left fade.
- Is the source file for these easy to put in source control too?

File: images/favicon/site.webmanifest
Comment (jameseaster) [reply images/favicon/site.webmanifest:1]: Updated to be indented like normal JSON: aea6397

Comment (jameseaster) [reply images/favicon/site.webmanifest:1]: @BigBlueHat Any thoughts on better placement for the web.manifest file? It's directly correlated with the favicon which is why they are grouped together here (not saying that's the best solution, just the current reasoning).

Comment (BigBlueHat) [reply images/favicon/site.webmanifest:1]: I'd put `site.webmanifest` at the root of the site and fix the paths. I know the way we're using it is favicon focused, but it's technically "about" the whole site, so putting it at the root makes the most sense to me (and is pretty typical).

**Comment by BigBlueHat** [comment]
> * The white on light blue contrast makes it more difficult to read at small sizes, especially with the light upper left fade.

I agree the contrast is probably not strong enough when it's shrunk down super tiny. @jameseaster maybe push the gradient a bit so there's more blue than white?

File: images/favicon/site.webmanifest
Comment (jameseaster) [reply images/favicon/site.webmanifest:1]: @BigBlueHat @davidlehn

Update: site.webmanifest is now at the root level and the favicon is a solid blue color that matches canivc's primary theme color. 3ab3530

File: site.webmanifest
Comment (davidlehn) [inline site.webmanifest:1]: Need an .eleventy.js rule to copy this.

Comment (jameseaster) [reply site.webmanifest:1]: Totally, good catch, thanks!

**Review by davidlehn — APPROVED** [review]

**Review by BigBlueHat — APPROVED** [review]

**Pull request merged**